controller/script/destination-client/main.go

```diff
@@ -18,12 +18,12 @@ import (
 func main() {
 	rand.Seed(time.Now().UnixNano())
 
-	addr := flag.String("addr", ":8089", "address of destination service")
+	addr := flag.String("addr", ":8086", "address of destination service")
```

Comment:
"address of destination service" => "address of the proxy API" or something similar? On a related note, maybe we should rename this directory from `destination-client` to `proxy-api-client`?

Reply:
I've gone back and forth on this.  You can think of this as a client of the the destination gRPC service, which is served by the proxy-api process.  So I can see it either way.  What do you think?